src/wp-admin/includes/plugin-install.php

```diff
@@ -533,7 +533,7 @@ function install_plugin_information() {
 	);
 
 	if ( is_wp_error( $api ) ) {
-		wp_die( $api );
+		wp_die( $api, 538 );
```

Comment:
I'm unfamiliar with 538 and can't find it in any documentation. Can you provide more context?